### PR TITLE
internal/fix-discojuice-issues

### PR DESCRIPTION
**Pull request by milanmajchrak**
| Phases            | MP | MM  |  MB  | MR |   JM | Total  |
|-----------------|----:|----:|----:|-----:|-----:|-------:|
| ETA                  |  0  |  0  |    0 |     0 |      0 |        0 |
| Developing      |  0  |  0.2  |    0 |    0 |      0 |         0 |
| Review             |  0  |  0  |    0 |    0 |      0 |         0 |
| Total                |   -  |   -  |   -   |  -    |   -    |         0 |
| ETA est.             |      |      |       |       |         |         0 |
| ETA cust.           |   -  |   -  |   -  |   -   |   -     |        0 |

### CODE REVIEW

File: dspace-server-webapp/src/main/java/org/dspace/app/rest/ClarinDiscoJuiceFeedsDownloadService.java [hunk 1]
```diff
@@ -269,7 +269,7 @@ private static JSONArray downloadJSON(String url) {
             Object obj = parser.parse(new InputStreamReader(conn.getInputStream()));
             return (JSONArray) obj;
         } catch (IOException | ParseException e) {
-            log.error("Failed to obtain/parse " + url + "\nCheck timeouts, redirects, shibboleth config.\n" + e);
+            log.debug("ERROR: Failed to obtain/parse " + url + "\nCheck timeouts, redirects, shibboleth config.\n" + e);
```
Comment (MajoBerger) [inline dspace-server-webapp/src/main/java/org/dspace/app/rest/ClarinDiscoJuiceFeedsDownloadService.java:272]: I know why we do not log this as error. We want to reduce logs. But there is already reduction in cron job from each 2 minutes to each 2 hours. In prod environment, isn't this still an error?

**Review by MajoBerger — CHANGES_REQUESTED** [review]
please consider that one comment.

File: dspace-server-webapp/src/main/java/org/dspace/app/rest/ClarinDiscoJuiceFeedsDownloadService.java (hunk 1 above)
Comment (milanmajchrak) [reply dspace-server-webapp/src/main/java/org/dspace/app/rest/ClarinDiscoJuiceFeedsDownloadService.java:272]: Yeah I forgot that, thank you

**Review by MajoBerger — CHANGES_REQUESTED** [review]
where error

**Review by MajoBerger — APPROVED** [review]

**Comment by milanmajchrak** [comment]
> where error

that's not in the PRs changes

**Review by mrovnanik — APPROVED** [review]

**Pull request merged**